core/src/main/java/com/linecorp/armeria/internal/server/annotation/AnnotatedService.java

```diff
@@ -100,6 +100,9 @@
 
     private static final MethodHandles.Lookup lookup = MethodHandles.lookup();
 
+    private static final CompletableFuture<AggregatedResult>
+            NO_AGGREGATION_FUTURE = CompletableFuture.completedFuture(AggregatedResult.EMPTY);
```

Comment:
[`dd6377c` (#4133)](https://github.com/line/armeria/pull/4133/commits/dd6377c21b45c99e8e3e51681d82f52c8b1808e3#diff-c20e52c324b0324d64250cbff999c20d229cf437f6b90e3410457335a2ef9f1f)
A new Checkstyle rule is expected to introduce.